Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM pytorch/pytorch:2.0.1-cuda11.7-cudnn8-devel
```

Comment:
@pablospe When you're trying to get it work without an additional conda (since conda is already in this base image), you can also change the base image to `runtime` instead of `devel`, that was an oversight.